### PR TITLE
Update with new data releases

### PR DESCRIPTION
**Pull request by markberger**
I've added an `environment.yaml` file and generated the updated files by running `code/run.sh`.  I'm still fairly new to anaconda but I believe I've only included the necessary packages and their dependencies (networkx, pandas, obonet, and requests). Note that some of the files for taxa are now deleted because they are not in the latest version of the files.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+name: gene-ontology
+channels:
+  - defaults
```
Comment (dhimmel) [inline environment.yml]: change the channel to conda-forge

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  - libffi=3.2.1=hd88cf55_4
+  - libgcc-ng=8.2.0=hdf63c60_1
+  - libgfortran-ng=7.3.0=hdf63c60_0
+  - libstdcxx-ng=8.2.0=hdf63c60_1
+  - mkl=2019.0=118
+  - mkl_fft=1.0.6=py36h7dd41cf_0
+  - mkl_random=1.0.1=py36h4414c95_1
+  - ncurses=6.1=hf484d3e_0
+  - networkx=2.2=py36_1
+  - numpy=1.15.3=py36h1d66e8a_0
```
Comment (dhimmel) [inline environment.yml]: Remove build numbers for all packages, i.e. 

```
- conda-forge::numpy=1.15.3
```

I also usually specify the channel as done with `conda-forge::`.

The build numbers have the effect of making envs restricted to a single operating system (e.g. mac or linux).

It's also probably better to only specify the versions of packages that we explicitly import, such as `pandas` but not behind-the-scenes packages that apply specially to certain architectures such as `intel-openmp`.

See the example at https://github.com/greenelab/manubot-rootstock/blob/dcfe402544f90e0047a247cdd8d01a2bca08df54/build/environment.yml

File: environment.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    - simplegeneric==0.8.1
+    - terminado==0.8.1
+    - testpath==0.4.2
+    - tornado==5.1.1
+    - traitlets==4.3.2
+    - urllib3==1.24
+    - wcwidth==0.1.7
+    - webencodings==0.5.1
+    - widgetsnbextension==3.4.2
+prefix: /opt/modules/pkgs/anaconda/3.6/envs/gene-ontology2
```
Comment (dhimmel) [inline environment.yml]: Remove prefix as this is device / user specific.

**Review by dhimmel — CHANGES_REQUESTED** [review]
I think it'd be best to split the environment PR from the data update PR. Should we just revert the environment.yml change from this PR... then I think the data update is ready to merge.

Then in a subsequent PR we can do the the environment update. I don't want to take too much of your time, so let me know if you're reaching the max time you've got for this, or whether you're able to keep going with enhancing the environment spec.

**Comment by dhimmel** [comment]
> Note that some of the files for taxa are now deleted because they are not in the latest version of the files.

Great. glad that worked

**Comment by markberger** [comment]
I've dropped `environment.yaml` from this commit so that it is just the data update. I can make the changes to the spec and test them out but it probably won't be for a few days.

**Review by dhimmel — APPROVED** [review]
Great. Thanks for the update. Always exciting to see a community contribution like this to help keep things up to date (although ideally we'd automate the updates with continuous analysis).

**Comment by dhimmel** [comment]
> . I can make the changes to the spec and test them out but it probably won't be for a few days

Not a problem. Feel free to open a WIP progress with the old env if you'd like, such that if you don't have time we can always use that as a starting point!

**Pull request merged**